src/pages/components.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Header } from "../../components/components/header";
+import { NavigationBar } from "../../components/components/navigation_bar";
```

Comment:
Keep your file name as NavigationBar instead of navigation_bar

Reply:
And remove one components from file structure, currently it is  'components/components'

Reply:
Keep you file structure as below .
components 
     -> componentList
          ComponentList.tsx
          Component.tsx
     -> componentCreationForm
          ComponentCreationForm.tsx
          components(will contains all fields used in ComponentCreationForm)

Reply:
for file structuring, it is components/components/, keeping in mind the incident views